src/toolchain/toolchain.ts

```diff
@@ -181,12 +182,84 @@ export class SwiftToolchain {
      * @returns Folders for each Xcode install
      */
     public static async getXcodeInstalls(): Promise<string[]> {
+        if (process.platform !== "darwin") {
```

Comment:
on line 150 the method `getXcodeDeveloperDir` calls `xcode-select -p`. If I choose another Xcode as my toolchain from within VSCode this still points to my system default toolchain.